app/views/admin/basic_projects/_basic_projects_grid_definition.html.slim

```diff
@@ -1,5 +1,5 @@
 = define_grid(@basic_projects_grid, hide_csv_button: true,
-    html: {class: 'rowlink table-hover', data: {link: @csv_mode ? nil : 'row'}}) do |g|
+    html: {class: 'rowlink table-hover', data: {link: 'row'}}) do |g|
```

Comment:
This condition has no impact on successful exporting to CSV. Data link information does not show up in the table when exporting.